src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+        action="#"
+        method="post"
+        class="form"
+        data-qa="big"
+      >
```

Comment:
Pay attention on code style: It should be 2 spaces and closed bracket should be on the same line as open
```suggestion
    <form
      action="#"
      method="post"
      class="form"
      data-qa="big"
    >
```

Reply:
Thanks done